src/app/page.tsx

```diff
@@ -3,11 +3,15 @@ import { Footer } from '../components/Footer';
 import NewsletterForm from '../components/NewsletterForm';
 import { Map } from '../components/Map';
 import { Team } from '../components/Team';
+import Image from 'next/image';
 
 export default function Home() {
   return (
     <>
       <section className="left">
+        <div class="illustration-home">
+          <Image src="/images/illustration-directions.svg" width="80" height="80" alt="" />
```

Comment:
can you add a real "alt" for accessibility?

Reply:
Done.